views/config_postProcessing.mako

```diff
@@ -200,6 +200,18 @@
                                     <span class="component-desc"><b>NOTE:</b> Only working with RAR archive</span>
                                 </label>
                             </div>
+                            <div id="content_unpack">
```

Comment:
No @fernandog you missed my review comment. `content_unpack` should be `content-unpack`.